s5_subwords/local/sw_methods/bpe/prepare_subword_text.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# 2019 Dongji Gao
```

Comment:
What license is this file released under? Apache 2.0? MIT?

Reply:
Apache 2.0 I would guess as it is stated in the run file they created orignally

Reply:
ok can you add that to the file?